Formula/kibana@6.rb

```diff
@@ -14,7 +14,7 @@ class KibanaAT6 < Formula
 
   keg_only :versioned_formula
 
-  deprecate! date: "2022-02-10", because: :unsupported
+  disable! date: "2022-09-06", because: :unsupported
```

Comment:
Should this match `node@10`, which was disabled on `2022-07-31`? `kibana` was also disabled then. This would allow `node@10` and dependents to be removed at same time.

Reply:
Yea, probably. Will change it.